employees.rb

```diff
@@ -0,0 +1,50 @@
+require './departments'
+
+class Employee
+
+  attr_reader :name, :salary, :reviews, :satisfactory
+
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @reviews = []
+  end
+
+  def employee_review(string)
+    @reviews << string
+  end
+
+  def rated_unsatisfactory
+    @satisfactory = false
+  end
+
+  def rated_satisfactory
+    @satisfactory = true
+  end
+
+  def rated_satisfactory?
+    return @satisfactory
+  end
+
+  def give_raise(raise)
+    @salary = raise + @salary
+  end
+
+  def performance_from_review
+    poor_results = []
+    good_results = []
+    poor_performance = ["but", "not", "room for improvement", "difficult", "confusion", "less", "negative", "not", "lack", "inadequate", "limitation", "concern", "longer", "interrupt", "disagree", "off"]
+    good_performance = ["asset", "pleasure", "quick", "willing", "help", "success", "happy", "responsive", "consistent", "effective", "satisfied", "impressed", "enjoy", "devoted", "perfect", "ownership", "positive", "encourage", "well"]
+    if poor_results.include?(poor_performance)
+      poor_results << poor_performance
+    elsif good_results.include?(good_performance)
+      good_results << good_performance
+    end
+  end
```

Comment:
Your method ends here with this end, so the code below it isn't actually part of any method.  That may have been unintentional.